### PR TITLE
refine api doc

### PR DESCRIPTION
**Pull request by novain1102**
Re: https://github.com/cocos/cocos-engine/issues/13888

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/core/scheduler.ts [hunk 1]
```diff
@@ -35,19 +35,27 @@ const MAX_POOL_SIZE = 20;
...
 
 // data structures
 /**
  * @en A list double-linked list used for "updates with priority"
  * @zh 用于“优先更新”的列表
  * @class ListEntry
- * @param target not retained (retained by hashUpdateEntry)
+ * @param target
+ * @en target object, which is ISchedulable type, retained by hashUpdateEntry
+ * @zh 目标对象, 为ISchedulable类型. 被hashUpdateEntry持有
```
Comment (minggo) [inline cocos/core/scheduler.ts]: - Need `.` or `。` at the end of a sentence
- Should use capital character for at the begin of a sentence, so should change `target` to `Target`.

File: cocos/core/scheduler.ts [hunk 2]
```diff
@@ -35,19 +35,27 @@ const MAX_POOL_SIZE = 20;
...
+ * @en the priority
+ * @zh 优先级
  * @param paused
- * @param markedForDeletion selector will no longer be called and entry will be removed at end of the next tick
+ * @en whether is paused
+ * @zh 是否被暂停
+ * @param markedForDeletion
+ * @en mark for deletion. if true, selector will no longer be called and entry will be removed at end of the next tick
+ * @zh 删除标记, 当为true时, selector 将不再被调用，并且entry将在下一个tick结束时被删除
  */
```
Comment (minggo) [inline cocos/core/scheduler.ts:47]: I think parameters should be described in constructor, not in class description.

File: cocos/core/scheduler.ts [hunk 3]
```diff
@@ -86,13 +94,13 @@ class ListEntry {
...
  * @class HashUpdateEntry
- * @param list Which list does it belong to ?
- * @param entry entry in the list
- * @param target hash key (retained)
- * @param callback
+ * @param list @en Which list does it belong to @zh 所属的列表
+ * @param entry @en entry in the list @zh 所述的条目
+ * @param target @en hash key (retained) @zh 哈希键所对应的目标(被持有的)
+ * @param callback @en the callback function @zh 回调函数
  */
```
Comment (minggo) [inline cocos/core/scheduler.ts:108]: The same as above.

File: cocos/core/scheduler.ts [hunk 4]
```diff
@@ -237,7 +245,7 @@ class CallbackTimer {
         return true;
     }
     /**
-     * @return returns interval of timer
+     * @return @en returns interval of timer in seconds @zh 返回计时器的时间间隔, 以秒为单位
      */
     public getInterval () {
```
Comment (minggo) [inline cocos/core/scheduler.ts:258]: - return -> returns
- Should include function description and `returns` too.

File: cocos/core/scheduler.ts [hunk 5]
```diff
@@ -413,7 +423,8 @@ export class Scheduler extends System {
     /**
      * @en 'update' the scheduler. (You should NEVER call this method, unless you know what you are doing.)
      * @zh update 调度函数。(不应该直接调用这个方法，除非完全了解这么做的结果)
-     * @param dt delta time
+     * @param dt
+     * @en delta time @zh 更新间隔时间
```
Comment (minggo) [inline cocos/core/scheduler.ts]: What is unit?

File: cocos/core/scheduler.ts [hunk 6]
```diff
@@ -510,28 +521,28 @@ export class Scheduler extends System {
...
      * delay 值指定延迟时间，定时器会在延迟指定的时间之后开始计时，单位: 秒。
+     * 如果 paused 值为 true，那么直到 resume 被调用才开始计时。<br/>
      * @param callback
      * @param target
      * @param interval
      * @param [repeat]
      * @param [delay=0]
-     * @param [paused=fasle]
+     * @param [paused=false]
      */
```
Comment (minggo) [inline cocos/core/scheduler.ts:556]: missing description of parameters.

File: cocos/core/scheduler.ts (hunk 5 above)
Comment (novain1102) [reply cocos/core/scheduler.ts]: fixed.

File: cocos/core/scheduler.ts (hunk 6 above)
Comment (novain1102) [reply cocos/core/scheduler.ts:556]: fixed.

File: cocos/core/scheduler.ts (hunk 2 above)
Comment (novain1102) [reply cocos/core/scheduler.ts:47]: got it.

File: cocos/core/scheduler.ts (hunk 1 above)
Comment (novain1102) [reply cocos/core/scheduler.ts]: Got it. Done.

File: cocos/core/scheduler.ts [hunk 7]
```diff
@@ -411,9 +431,11 @@ export class Scheduler extends System {
     }
 
     /**
-     * @en 'update' the scheduler. (You should NEVER call this method, unless you know what you are doing.)
+     * @en 'update' the scheduler. (You should  delta time call this method, unless you know what you are doing.)
```
Comment (minggo) [inline cocos/core/scheduler.ts]: What's `delta time call`?

File: cocos/core/scheduler.ts [hunk 8]
```diff
@@ -507,31 +529,30 @@ export class Scheduler extends System {
...
-     * 指定回调函数，调用对象等信息来添加一个新的定时器。<br/>
-     * 如果 paused 值为 true，那么直到 resume 被调用才开始计时。<br/>
-     * 当时间间隔达到指定值时，设置的回调函数将会被调用。<br/>
-     * 如果 interval 值为 0，那么回调函数每一帧都会被调用，但如果是这样，
-     * 建议使用 scheduleUpdateForTarget 代替。<br/>
-     * 如果回调函数已经被定时器使用，那么只会更新之前定时器的时间间隔参数，不会设置新的定时器。<br/>
-     * repeat 值可以让定时器触发 repeat + 1 次，使用 `macro.REPEAT_FOREVER`
-     * 可以让定时器一直循环触发。<br/>
-     * delay 值指定延迟时间，定时器会在延迟指定的时间之后开始计时，单位: 秒。
+     * @en Specify the callback, target and other information to schedule a new timer
```
Comment (minggo) [inline cocos/core/scheduler.ts]: Missing `.` at the end of the sentence.

File: cocos/video/video-player.ts [hunk 9]
```diff
@@ -146,9 +146,9 @@ export class VideoPlayer extends Component {
 
     /**
      * @en
-     * The Video playback rate
+     * The Video playback rate. The value range is from (0.0 ~ 10.0).
```
Comment (minggo) [inline cocos/video/video-player.ts]: Range is [0,0 ~ 10.0]. `0.0` and `10.0` are included.

Comment (minggo) [reply cocos/video/video-player.ts]: Should check it.

File: cocos/video/video-player.ts [hunk 10]
```diff
@@ -165,9 +165,9 @@ export class VideoPlayer extends Component {
 
     /**
      * @en
-     * The volume of the video.
+     * The volume of the video. The value range is from (0.0 ~ 1.0).
      * @zh
-     * 视频的音量（0.0 ~ 1.0）
+     * 视频的音量（0.0 ~ 1.0）。
```
Comment (minggo) [inline cocos/video/video-player.ts]: The same as above.

File: cocos/video/video-player.ts [hunk 11]
```diff
@@ -165,9 +165,9 @@ export class VideoPlayer extends Component {
...
      */
     @slide
     @range([0.0, 1.0, 0.1])
```
Comment (minggo) [inline cocos/video/video-player.ts:173]: Is it a valid range?

File: cocos/video/video-player.ts [hunk 12]
```diff
@@ -299,24 +301,31 @@ export class VideoPlayer extends Component {
 
     /**
      * @en
-     * Raw video objects for user customization
+     * Get the original video object, generally used for user customization.
```
Comment (minggo) [inline cocos/video/video-player.ts]: Get -> Gets

File: cocos/video/video-player.ts [hunk 13]
```diff
@@ -299,24 +301,31 @@ export class VideoPlayer extends Component {
...
+     * 获取原始视频对象，一般用于用户定制。
      */
     get nativeVideo () {
         return (this._impl && this._impl.video) || null;
     }
 
     /**
      * @en
-     * The current playback time of the now playing item in seconds, you could also change the start playback time.
+     * Get the time progress of the current video playback.
```
Comment (minggo) [inline cocos/video/video-player.ts]: The same as above.

File: cocos/video/video-player.ts [hunk 14]
```diff
@@ -299,24 +301,31 @@ export class VideoPlayer extends Component {
...
+     * 获取当前视频播放的时间进度。
      */
     get currentTime () {
         if (!this._impl) { return this._cachedCurrentTime; }
         return this._impl.getCurrentTime();
     }
+
+    /**
+     * @en
+     * Set the time point when the video starts to play, in seconds.
```
Comment (minggo) [inline cocos/video/video-player.ts]: The same as above.

File: cocos/web-view/web-view.ts [hunk 15]
```diff
@@ -90,17 +89,17 @@ export class WebView extends Component {
...
+     * 原始网页对象，用于用户定制。
      */
     get nativeWebView () {
         return (this._impl && this._impl.webview) || null;
     }
 
     /**
      * @en
-     * Get current audio state.
+     * Get the current webview state.
```
Comment (minggo) [inline cocos/web-view/web-view.ts]: Get -> Gets.

File: cocos/web-view/web-view.ts [hunk 16]
```diff
@@ -111,7 +110,7 @@ export class WebView extends Component {
 
     /**
      * @en
-     * Set javascript interface scheme (see also setOnJSCallback). <br>
+     * Set javascript interface scheme (see also setOnJSCallback).
```
Comment (minggo) [inline cocos/web-view/web-view.ts]: The same as above.

File: cocos/video/video-player.ts (hunk 9 above)
Comment (novain1102) [reply cocos/video/video-player.ts]: fixed.

File: cocos/video/video-player.ts (hunk 11 above)
Comment (novain1102) [reply cocos/video/video-player.ts:173]: fixed.

File: cocos/video/video-player.ts (hunk 13 above)
Comment (novain1102) [reply cocos/video/video-player.ts]: fixed.

File: cocos/video/video-player.ts (hunk 14 above)
Comment (novain1102) [reply cocos/video/video-player.ts]: fixed.

File: cocos/web-view/web-view.ts (hunk 16 above)
Comment (novain1102) [reply cocos/web-view/web-view.ts]: fixed.

File: cocos/web-view/web-view.ts (hunk 15 above)
Comment (novain1102) [reply cocos/web-view/web-view.ts]: fixed.

File: cocos/video/video-player.ts [hunk 17]
```diff
@@ -146,9 +146,9 @@ export class VideoPlayer extends Component {
 
     /**
      * @en
-     * The Video playback rate. The value range is from (0.0 ~ 10.0).
+     * The Video playback rate. The value range is from [0.0 ~ 10.0].
      * @zh
-     * 视频播放时的速率, 值的区间为（0.0 ~ 10.0）。
+     * 视频播放时的速率, 值的区间为[]0.0 ~ 10.0]。
```
Comment (minggo) [inline cocos/video/video-player.ts]: `[]0.0 ~ 10.0]` -> `[0.0 ~ 10.0]`

**Review by minggo — APPROVED** [review]

**Pull request merged**